app/Http/Requests/ProductValidation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class ProductValidation extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'name' => ['required', 'max:255', Rule::unique('products')->ignore($this->product)],
+            'brand_id' => ['required', 'string', 'max:255'],
+            'color_id' => ['nullable', 'string', 'max:255'],
+            'size_id' => ['string', 'nullable', 'max:255'],
```

Comment:
Where is the Exists check? max rule not needed

Reply:
@jayesh-parmar Please read the full like again.

You may contact @rajan-parmar if not sure.